llvm/lib/Target/RISCV/RISCVSchedule.td

```diff
@@ -22,6 +22,7 @@ def WriteIMul32     : SchedWrite;    // 32-bit multiply on RV64I
 def WriteJmp        : SchedWrite;    // Jump
 def WriteJal        : SchedWrite;    // Jump and link
 def WriteJalr       : SchedWrite;    // Jump and link register
+def WriteJmpReg     : SchedWrite;    // Jump register
```

Comment:
which instruction uses this?

Reply:
Removed in https://github.com/llvm/llvm-project/pull/119885. `WriteJalr` should be used.